catalog-rest-service/src/main/java/org/openmetadata/catalog/exception/CatalogExceptionMessage.java

```diff
@@ -41,6 +41,10 @@ public static String entityTypeNotFound(String entity) {
     return String.format("Entity type %s not found", entity);
   }
 
+  public static String fieldIsNull(String entity) {
```

Comment:
field instead of entity for variable name?

Reply:
Yep